src/components/Breadcrumb/Breadcrumb.vue

```diff
@@ -0,0 +1,197 @@
+<!--
+ - @copyright Copyright (c) 2020 Raimund Schlüßler <raimund.schluessler@mailbox.org>
+ -
+ - @author Raimund Schlüßler <raimund.schluessler@mailbox.org>
+ -
+ - @license GNU AGPL version 3 or any later version
+ -
+ - This program is free software: you can redistribute it and/or modify
+ - it under the terms of the GNU Affero General Public License as
+ - published by the Free Software Foundation, either version 3 of the
+ - License, or (at your option) any later version.
+ -
+ - This program is distributed in the hope that it will be useful,
+ - but WITHOUT ANY WARRANTY; without even the implied warranty of
+ - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ - GNU Affero General Public License for more details.
+ -
+ - You should have received a copy of the GNU Affero General Public License
+ - along with this program. If not, see <http://www.gnu.org/licenses/>.
+ -
+ -->
+
+<docs>
+
+### General description
+
+This component is meant to be used inside a Breadcrumbs component.
+
+</docs>
+
+<template>
+	<div ref="crumb"
+		class="crumb"
+		:class="{over: hovering}"
+		draggable="false"
+		@dragstart.prevent="() => {/** Prevent the breadcrumb from being draggable. */}"
+		@drop.prevent="dropped"
+		@dragover.prevent="() => {}"
+		@dragenter="dragEnter"
+		@dragleave="dragLeave">
+		<element :is="tag" :to="to" :href="href">
+			<span v-if="icon" :class="icon" class="icon" />
+			<span v-else>{{ title }}</span>
+		</element>
+		<Actions :force-menu="forceMenu">
```

Comment:
ALignment is not ideal, I would reduce the right padding to 2 or 3px for the link.
![Capture d’écran_2020-03-28_21-49-11](https://user-images.githubusercontent.com/14975046/77833484-faa0b000-713d-11ea-98ea-2fcdbb328d0d.png)
___


![Peek 28-03-2020 21-47](https://user-images.githubusercontent.com/14975046/77833481-eb216700-713d-11ea-8dd9-7485f041c890.gif)
![Capture d’écran_2020-03-28_21-48-01](https://user-images.githubusercontent.com/14975046/77833482-eceb2a80-713d-11ea-9ced-525b8d9c938b.png)